src/js/components/Layer/doc.js

```diff
@@ -20,8 +20,18 @@ export const doc = Layer => {
 
   DocumentedLayer.propTypes = {
     animate: PropTypes.bool
-      .description('Animation transition of the Layer content when it opens.')
+      .description(
+        `Whether to animate the Layer content when it opens. This
+        property is deprecated and will be removed in the next major version
```

Comment:
Just so we'll be on the same page, the prop `animate` is being used in both Collapsible and Accordion components, should we incline to make it aligned with the prop `animation` instead, or should it be OK because `animate` performs only as a boolean in those cases (Collapsible and Accordion).

Reply:
Good question. I switched to `animation` to align with `Box`. Since I wanted the ability to add types of animations, `animate` seems less consistent. I could see eventually enhancing Collapsible and Accordion in similar ways eventually. But, I don't know that those changes are needed at the moment.